src/custom/components/Tokens/TokensTableRow.tsx

```diff
@@ -42,6 +56,8 @@ const DataRow = ({ tokenData, index }: DataRowParams) => {
           </HideMedium>
         </LinkWrapper>
       </Label>
+
+      <BalanceValue hasBalance={!!hasBalance}>{formattedBalance}</BalanceValue>
```

Comment:
Can you add the full amount on hover? (as title text)

Reply:
Done